test/unittest/xmlparser/XMLElementParserTests.cpp

```diff
@@ -840,7 +851,8 @@ TEST_F(XMLParserTests, getXMLPortParameters_NegativeClauses)
                     "\
                     <port>\
                         <portBase>" + parameters[0] + "</portBase>\
-                        <domainIDGain>" + parameters[1] + "</domainIDGain>\
+                        <domainIDGain>" + parameters[1] +
+                    "</domainIDGain>\
```

Comment:
I think it was easier to read before. Uncrustify does not complain for me
```suggestion
                        <domainIDGain>" + parameters[1] + "</domainIDGain>\
```

Reply:
9507762